src/service.js

```diff
@@ -366,7 +366,6 @@ function getServiceInternal(holder, id) {
     devAssert(s.context, `Service ${id} registered without context.`);
     s.obj = new s.ctor(s.context);
     devAssert(s.obj, `Service ${id} constructed to null.`);
-    s.ctor = null;
```

Comment:
It's not necessary to keep the ctor around. This actually didn't free up any memory, anyways, since the `s.obj` is an instance of the `ctor`, and `s.obj.__proto__ === ctor.prototype` and `s.obj.constructor === ctor` (`ctor` can never be GCd).

Reply:
Sometimes those are factories. E.g. `function() {... return new Ctor()}`. But I think your point still stands. Unlikely big winnings.